tests/test_validate.py

```diff
@@ -890,3 +890,16 @@ def test_containsnoneof_mixing_types():
 
     with pytest.raises(ValidationError):
         validate.ContainsNoneOf([1, 2, 3])((1,))
+
+
+def test_and():
+    def is_even(value):
+        if value % 2 != 0:
+            raise ValidationError("Not an even value.")
+
+    validator = validate.And([validate.Range(min=0), is_even])
```

Comment:
What about adding a factory to create such `validator.And` without combining sub-validators into a list?
Before:
```python
    validator = validate.And([validate.Range(min=0), is_even])
```
After:
```python
    validator = validate.and(validate.Range(min=0), is_even, is_magic)
```
?

Reply:
It's not only about the number of characters, but also about visual perception and the forced creation of an optimal sequence - tuple.

In addition, this will not allow to create "And" without validators.

Reply:
OK. Seems fine since it's a fairly common pattern in Python. Added in my latest commit.

> this will not allow to create "And" without validators.

How?

Reply:
Is there any reason not to make the *args syntax the default? It is easy to `(*spread)` an iterable into args when we use it internally, but as a user API, *args looks prettier. I doubt users are going to have their validators in a list for some other use.

Reply:
@deckar01 By `default`, do you mean using `validate.and_` in `Field._validate_all`? Should we make `validate.And` private (`validate._And`)?

Reply:
I mean `And.__init__(*validators, error=None)` and no `_and`. It would be nice to minimize the API surface area. When we use it internally we can just as easily call `And(*validators, error=error)`.

Reply:
Good point; not much benefit in having 2 ways to do things here.